Examples/Histogram/Parser.go

```diff
@@ -30,6 +33,11 @@ func main() {
 	fmt.Println(histmap)
 	var ht Export
 	ht.Field1 = histmap
-	result := prometheus_handler.GenericPromDataParser(ht)
+	addToHandler = append(addToHandler, struct {
+		Structure interface{}
+		Datatype  string
+		Labelname string
```

Comment:
Datatype => DataType
Labelname => LabelName 

minor suggestions.. we should follow camel case style.